.github/pull_request_template.md

```diff
@@ -16,7 +16,36 @@
 - [ ] Add a description of your pull request and instructions for the reviewer to verify your work.
 - [ ] If your pull request is for a specific ticket, link to it in the description.
 - [ ] Stay on point and keep it small so the merge request can be easily reviewed.
-- [ ] Consider adding unit tests, especially for bug fixes. If you don't, tell us why.
 - [ ] Tests and linting passes.
-- [ ] Consider updating documentation. If you don't, tell us why.
-- [ ] If relevant, list the environments / browsers in which you tested your changes.
+
+#### Unit tests
+
+- [ ] Not required
+- [ ] Added
+
+#### Documentation
+
+- [ ] Updated
+
+#### Browser testing
+
+- [ ] I have tested in the following browsers and environments:
+
+#### Data protection
+
+- [ ] Not relevant
+- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly
+
+#### Accessibility
+
+- [ ] Automated WCAG 2.1 tests pass
```

Comment:
**Question**: Do we plan on having this as part of CI?

Reply:
Yes it would be nice to do that - I will add a ticket for it.

Reply:
(I don't know if we've done it in github actions before)